packages/manager/apps/pci-savings-plan/src/components/LegalLinks/LegalLinks.test.tsx

```diff
@@ -44,8 +40,8 @@ describe('LegalLinks', async () => {
     render(<LegalLinks />);
 
     await waitFor(() => {
-      expect(screen.getByText(MOCK_CONTRACTS[0].name)).toBeInTheDocument();
-      expect(screen.getByText(MOCK_CONTRACTS[1].name)).toBeInTheDocument();
+      expect(screen.getByTestId(MOCK_CONTRACTS[0].name)).toBeInTheDocument();
+      expect(screen.getByTestId(MOCK_CONTRACTS[1].name)).toBeInTheDocument();
```

Comment:
I don't understand this modification, we should avoid test id when possible

Also waitFor don't seem necessary here